mmengine/visualization/visualizer.py

```diff
@@ -7,6 +7,7 @@
 import numpy as np
 import torch
 import torch.nn.functional as F
+from matplotlib.font_manager import FontProperties
```

Comment:
Suggest moving this import clause to `def draw_texts(xxx)`. More details at https://github.com/open-mmlab/mmengine/pull/837

Reply:
I thought about that, but draw_texts() argument type declaration will use FontProperties.

Reply:
You can use TYPE_CHECKING like this:

https://github.com/open-mmlab/mmengine/blob/50465a20022938444f90314e9ce217607d8c215c/mmengine/visualization/utils.py#L9

Reply:
> You can use TYPE_CHECKING like this:
> 
> https://github.com/open-mmlab/mmengine/blob/50465a20022938444f90314e9ce217607d8c215c/mmengine/visualization/utils.py#L9

Done~

Reply:
> You can use TYPE_CHECKING like this:
> 
> https://github.com/open-mmlab/mmengine/blob/50465a20022938444f90314e9ce217607d8c215c/mmengine/visualization/utils.py#L9

i find there is a difference between the two cases:
not only argument type declaration will use, but also check_type_and_length() and value2list() will use FontProperties
if use TYPE_CHECKING,  https://github.com/KevinNuNu/mmengine/blob/ee500de0460942c40e65683f6a56588a78979e1c/mmengine/visualization/visualizer.py#L504-L510 will raise a NameError: name 'FontProperties' is not defined

Reply:
Need to import `FontProperties` in `draw_texts` method.

Reply:
> Need to import `FontProperties` in `draw_texts` method.

you mean import `FontProperties` in `draw_texts` method and use TYPE_CHECKING  at the same time？

Reply:
> Need to import `FontProperties` in `draw_texts` method.

You mean import `FontProperties` in `draw_texts` method and use TYPE_CHECKING  at the same time？

Reply:
ok, i know how to do it